test/e2e/tests/tokens/increase-token-allowance.spec.js

```diff
@@ -296,6 +293,8 @@ describe('Increase Token Allowance', function () {
       css: '.transaction-list__completed-transactions .activity-list-item [data-testid="activity-list-item-action"]',
       text: 'Increase TST spending cap',
     });
+
+    await driver.delay(2000);
```

Comment:
Is there an alternative possible?  What are we waiting for here?

Reply:
Another alternative would be to check if the status in the extension is updated, which is already mentioned above the delay. However we still require a small delay, so it should be fine.